test/collaboration.test.js

```diff
@@ -331,8 +348,8 @@ describe('slate operations propagate between editors', () => {
       // Verify initial states.
       const inputAsJSON = input.map(nodeToJSON);
       expect(src.slateDoc.document.nodes.toArray().map(nodeToJSON)).toStrictEqual(inputAsJSON);
-      expect(toSlateDoc(src.syncDoc).map(nodeToJSON)).toStrictEqual(inputAsJSON);
-      expect(toSlateDoc(dst.syncDoc).map(nodeToJSON)).toStrictEqual(inputAsJSON);
+      expect(toSlateDoc(src.syncDoc.get('document')).map(nodeToJSON)).toStrictEqual(inputAsJSON);
```

Comment:
The "actual" values being passed to expect() here are getting quite complicated (they were already pretty bad in the state I had left things, so definitely largely my fault).

Not necessarily for this PR, but perhaps in a followup -- we could probably extract these patterns into simple helper functions that would make this code (and the later "Verify final states") code much easier to read.

Reply:
Agreed!